cve_bin_tool/cli.py

```diff
@@ -82,7 +82,9 @@
 
 
 class StringToListAction(argparse.Action):
+    """Convert comma-separated strings to lists."""
```

Comment:
```suggestion
    """Convert comma-separated strings to lists."""

```

Reply:
Black is complaining about a formatting nitpick.  I'm pretty sure it's a missing line here.  (you can check yourself by running `black cli.py` and it'll fix whatever it thinks needs fixing)